apps/parse-rated-paragraphs/src/app/parse-rated.ts

```diff
@@ -0,0 +1,56 @@
+import * as fs from 'fs';
+import {
+  ParsedArticleParagraphStandalone,
+  ArticleParagraphBacksUpClaim,
+} from '@foodmedicine/interfaces';
+
+const allParagraphsBasePath = './tmp/correlated-paragraphs/';
+function getAllJsonPaths(): string[] {
+  return fs.readdirSync(allParagraphsBasePath);
+}
+
+function getParagraphsFromFile(
+  filename: string
+): Promise<ParsedArticleParagraphStandalone[]> {
+  return new Promise((res, rej) => {
+    fs.readFile(`${allParagraphsBasePath}/${filename}`, (err, data) => {
+      if (err) {
+        rej(err);
+      }
+      const json = JSON.parse(data.toString());
+      res(json as ParsedArticleParagraphStandalone[]);
+    });
+  });
+}
+
+function paragraphIsRated(
+  paragraph: ParsedArticleParagraphStandalone
+): boolean {
+  return paragraph.backsUpClaim !== ArticleParagraphBacksUpClaim.notApplicable;
+}
+
+function saveParagraphs(
+  path: string,
+  paragraphs: ParsedArticleParagraphStandalone[]
+) {
+  fs.writeFileSync(path, JSON.stringify(paragraphs));
+}
+
+// TODO the return value will be changed with the implementation of the frontend
+function createNewPath(originalFileName: string): string {
+  return `${__dirname}/../../../tmp/rated-paragraphs/${originalFileName}`;
+}
+
+export async function storeRatedParagraphs() {
+  const paragraphFilenames = getAllJsonPaths();
+  console.log(paragraphFilenames);
```

Comment:
remove console.logs?  These are good for temporary testing but we generally try not to console.log in final products often